src/Microsoft.HttpRepl.Telemetry/Microsoft.HttpRepl.Telemetry.csproj

```diff
@@ -7,4 +7,13 @@
     <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" />
     <PackageReference Include="Microsoft.Win32.Registry" />
   </ItemGroup>
+
+  <ItemGroup>
+    <!--
+      This PackageReference is only here to deal with a Component Governance alert.
+      Once Microsoft.ApplicationInsights.WorkerService 2.22 is released, we can
+      update our reference to that and remove this.
+    -->
+    <PackageReference Include="System.Drawing.Common" />
```

Comment:
This should be unnecessary.  With CPM, versions pinned in the Directory.Packages.props file are applied to transitive dependencies as well.

Reply:
I didn't have transitive pinning enabled yet (its disabled by default). Added it, removed this PackageReference and a couple others that were just for transitive dependencies.

Reply:
Oh right, I forgot it's not the default (it was during the feature previews).  LGTM now ;)